extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -68,6 +68,12 @@ describes.realWin('amp-story', {
   beforeEach(() => {
     win = env.win;
     element = win.document.createElement('amp-story');
+    // element.getAmpDoc = () => env.ampdoc;
```

Comment:
@choumx @erwinmombay mocking this all seems to work, but seems odd to have to do